code-challenges.test.js

```diff
@@ -64,4 +114,18 @@ const vowelTester2 = "academy"
 const vowelTester3 = "challenges"
 // Expected output: 2
 
+// describe("vowelLog", () => {
+//     it("CCreate a function that takes in a string and logs the index of the first vowel", () => {
+        // const vowelTester1 = "learn"
+    //     expect(vowelTester1(object1)).toEqual(object1[1])
+//     })
+// })
 // b) Create the function that makes the test pass.
+
+const vowelLog = (object) => {
+    return object.search(/[aeiou]/g) 
```

Comment:
The 'g' flag in the regex is not necessary; the 'i' flag would be what you want for edge cases.